agasc/supplement/utils.py

```diff
@@ -43,8 +48,9 @@ def get_supplement_table(name, agasc_dir=None, as_dict=False):
     """
     agasc_dir = default_agasc_dir() if agasc_dir is None else Path(agasc_dir)
 
-    if name not in ('mags', 'bad', 'obs'):
-        raise ValueError("table name must be one of 'mags', 'bad', or 'obs'")
+    if name not in ('mags', 'bad', 'obs', 'last_updated', 'versions'):
```

Comment:
At this point it may be worth defining a module `AGASC_SUPPLEMENT_TABLES` constant with the names of all the available tables.

Reply:
Done